exchanges/binance/binance_test.go

```diff
@@ -2482,3 +2420,25 @@ func TestUFuturesHistoricalTrades(t *testing.T) {
 		t.Error(err)
 	}
 }
+
+func TestWsOrderExecutionReport(t *testing.T) {
+	payload := []byte(`{"stream":"jTfvpakT2yT0hVIo5gYWVihZhdM2PrBgJUZ5PyfZ4EVpCkx4Uoxk5timcrQc","data":{"e":"executionReport","E":1616627567900,"s":"BTCUSDT","c":"c4wyKsIhoAaittTYlIVLqk","S":"BUY","o":"LIMIT","f":"GTC","q":"0.00028400","p":"52789.10000000","P":"0.00000000","F":"0.00000000","g":-1,"C":"","x":"NEW","X":"NEW","r":"NONE","i":5340845958,"l":"0.00000000","z":"0.00000000","L":"0.00000000","n":"0","N":null,"T":1616627567900,"t":-1,"I":11388173160,"w":true,"m":false,"M":false,"O":1616627567900,"Z":"0.00000000","Y":"0.00000000","Q":"0.00000000"}}`)
+	err := b.wsHandleData(payload)
+	if err != nil {
+		t.Fatal(err)
```

Comment:
Does the failure of this part need to prevent the execution of the second part of this test? I feel like fatal should only be used for a prerequisite, say if one fails to make a currency pair before using it in a function